css-dev/burf-theme/layout/_footer.scss

```diff
@@ -334,6 +338,10 @@ body {
 		@extend %icon-twitter;
 	}
 
+	[href*="x.com"] {
```

Comment:
I don't see the x-alt version here but I also didn't see it for any of the other alt versions so we must be good

Reply:
yeah I don't think we need it for the social menus.